build.sc

```diff
@@ -27,40 +27,43 @@ val swamUrl = "https://github.com/satabin/swam"
 
 val swamDeveloper = Developer("satabin", "Lucas Satabin", "https://github.com/satabin")
 
-val fs2Version = "2.0.1"
+val fs2Version = "2.3.0"
 
-val pureconfigVersion = "0.12.1"
+val pureconfigVersion = "0.12.3"
 
 trait SwamModule extends ScalaModule with ScalafmtModule with Headers {
 
   def scalaVersion = "2.13.1"
 
   def scalacOptions =
-    Seq("-feature", "-deprecation", "-unchecked", "-Ypatmat-exhaust-depth", "off", "-Ywarn-unused:locals,imports", "-Ymacro-annotations")
+    Seq("-feature",
+        "-deprecation",
+        "-unchecked",
+        "-Ypatmat-exhaust-depth",
+        "off",
+        "-Ywarn-unused:locals,imports",
+        "-Ymacro-annotations")
```

Comment:
You might want to add `-Wvalue-discard`, it should have caught one of the things below

Reply:
true.